src/Internal.AspNetCore.Sdk/build/ApiCheck.targets

```diff
@@ -3,13 +3,32 @@
   for use outside of Microsoft.
 -->
 <Project>
-  <Target Name="ApiCheck" Condition=" '$(EnableApiCheck)' == 'true' ">
+  <Target Name="_GetApiListingFile">
     <PropertyGroup>
       <_ApiListingFileSuffix Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">netframework.json</_ApiListingFileSuffix>
       <_ApiListingFileSuffix Condition=" '$(_ApiListingFileSuffix)' == '' ">netcore.json</_ApiListingFileSuffix>
       <_ApiListingFilePath>$(MSBuildProjectDirectory)\baseline.$(_ApiListingFileSuffix)</_ApiListingFilePath>
-      <_ApiExclusionsFilePath>$(MSBuildProjectDirectory)\breakingchanges.$(_ApiListingFileSuffix)</_ApiExclusionsFilePath>
       <_ApiListingFile Condition=" Exists('$(_ApiListingFilePath)') ">$(_ApiListingFilePath)</_ApiListingFile>
+    </PropertyGroup>
+  </Target>
+
+  <Target
+      Name="Generate-ApiCheck-Baselines"
+      DependsOnTargets="_GetApiListingFile" >
```

Comment:
If there are any plans to add a KoreBuild target to generate baselines for all `src` projects in a repo, this target should have the same condition as the `ApiCheck` target.